src/module_ncio.f90

```diff
@@ -813,39 +812,39 @@ end subroutine close_dataset
   !> subroutine read_vardata(dset,varname,values,nslice,slicedim,errcode)
```

Comment:
The first line of the docblock is the brief sentence description. Remove: 
 subroutine read_vardata(dset,varname,values,nslice,slicedim,errcode)
From the documentation block, and place the documentation block directly before the subroutine that it documents.

So move this docblock to above read_vardata()

Reply:
modified following your comments, Thanks